src/Microsoft.FeatureManagement/ServiceCollectionExtensions.cs

```diff
@@ -60,7 +61,15 @@ public static IFeatureManagementBuilder AddFeatureManagement(this IServiceCollec
                 throw new ArgumentNullException(nameof(configuration));
             }
 
-            services.AddSingleton<IFeatureDefinitionProvider>(sp => new ConfigurationFeatureDefinitionProvider(configuration, sp.GetRequiredService<ILoggerFactory>()));
+            services.Configure<FeatureManagementOptions>(options =>
+            {
+                options.RequireFeatureManagementSection = false;
+            });
+
+            services.AddSingleton<IFeatureDefinitionProvider>(sp => new ConfigurationFeatureDefinitionProvider(
```

Comment:
I don't see a need for FeatureManagementOptions to be updated. In fact it's dangerous. If IFeatureDefinitionProvider that is created with the inline configuration is not the one that ends up being used (multiple addfeaturemanagmeent calls), then the options won't match the added service.

I think it needs to be

```
            services.TryAddSingleton<IFeatureDefinitionProvider>(sp =>
                new ConfigurationFeatureDefinitionProvider(
                    configuration, 
                    sp.GetRequiredService<ILoggerFactory>())
                {
                    UseTopLevelConfiguration = true
                });
```